hospitalresponse.cpp

```diff
@@ -0,0 +1,90 @@
+#include <stdio.h>
+#include <string.h>
+
+//input the name of the local emergency team
+//output total response time in seconds
+
+int main()
+{
+#define MAX_TEAMNAME_SIZE 100
+#define MAX_NUM_OF_TEAMS 100
+
+    char team_name[MAX_TEAMNAME_SIZE]; //initialisation of a string
+    int total_time; //Total response time
+    int total_min; //Total response time in minutes
+    int total_sec; //Total response time in seconds
+    int triage_time; //Time to identify patient medical issues
+    int assess_time; //Time to assess Patient
+    int assign_time;
+    int transfer_time;
+    int triage_min;
+    int triage_sec;
+    int assess_min;
+    int assess_sec;
+    int assign_min;
+    int assign_sec;
+    int transfer_min;
+    int transfer_sec;
+    int fastest_team[MAX_NUM_OF_TEAMS];
+
+    printf("\nTeam name: ");
+    // see https://en.cppreference.com/w/c/io/fscanf
+    // always specify a max size and char * arrays should not have a pointer taken.
+    scanf("%99s", team_name); //INPUT (Data Type,Variable that input is assigned to)

```

Comment:
you can't use the define or constexpr in the string unless you construct the format dynamically.